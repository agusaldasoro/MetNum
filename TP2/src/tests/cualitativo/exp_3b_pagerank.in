0 0.85 0 tests/cualitativo/exp_3b.txt 0.00001 
