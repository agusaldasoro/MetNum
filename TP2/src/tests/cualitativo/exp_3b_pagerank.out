0.089374
0.198877
0.140997
0.070095
0.042494
0.042669
0.029450
0.008320
0.056371
0.008320
0.008320
0.063624
0.008320
0.008320
0.008320
0.126684
0.009498
0.041868
0.038082
